utils/gear-runtime-test-cli/Cargo.toml

```diff
@@ -15,24 +15,34 @@ serde_json = "1.0.82"
 quick-xml = { version = "0.23", features = [ "serialize" ] }
 rayon = "1.5"
 
-# Substrate deps
+# Substrate
 frame-support = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false }
 frame-system = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
 sc-cli = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
 sp-core = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false }
 sc-service = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
 sp-io = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false }
 sp-runtime = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false }
+pallet-aura = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false, optional = true }
 pallet-balances = { version = "4.0.0-dev", default-features = false, git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
-sp-consensus-aura = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", default-features = false }
+sp-consensus-slots = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
+sp-consensus-aura = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", optional = true }
+sp-consensus-babe = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", optional = true }
+sp-finality-grandpa = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable", optional = true }
 
-# Internal deps
+# Gear
 gear-common = { path = "../../common" }
 gear-core = { path = "../../core" }
-gear-runtime = { path = "../../runtime", features = ["debug-mode"] }
+gear-runtime = { path = "../../runtime/gear", optional = true, features = ["debug-mode"] }
+vara-runtime = { path = "../../runtime/vara", optional = true, features = ["debug-mode"] }
 gear-test = { path = "../../gear-test" }
 pallet-gear = { path = "../../pallets/gear" }
 pallet-gear-gas = { path = "../../pallets/gas" }
 pallet-gear-debug = { path = "../../pallets/gear-debug" }
 gear-core-processor = { path = "../../core-processor" }
 junit-common = { path = "../junit-common" }
+
+[features]
+gear-native = ["gear-runtime", "sp-consensus-aura", "authoring-aura"]
+vara-native = ["vara-runtime", "sp-consensus-babe", "sp-finality-grandpa"]
+authoring-aura = []
```

Comment:
do we need separate flag for aura, if gear uses aura and vara uses babe?

Reply:
It's helpful to highlight mutually exclusive features. That is if both `gear-native` and `vara-native` are enabled we use `aura` and avoid importing the same definitions from conflicting packages. In some cases there's no conflict though